packages/serverless/src/domain/use-case/schema.ts

```diff
@@ -166,7 +166,7 @@ export async function getSchemaDefinition(
     (fieldName) =>
       ({
         name: schema[fieldName].name,
-        kind: schema[fieldName].king,
```

Comment:
Please make sure that this format is correct. I don't think this format is good for front-end.

Reply:
Removed